dts/bindings/display/solomon,ssd1306fb-common.yaml

```diff
@@ -48,3 +48,10 @@ properties:
       The RESET pin of SSD1306 is active low.
       If connected directly the MCU pin should be configured
       as active low.
+
+  boot-delay-ms:
```

Comment:
I wonder if this really belongs to DT or Kconfig, because of the way it is described. If part of hw description, maybe it should be named more like `ready-time-ms`, and described as "time it takes for the device to become ready/responsive", then, drivers can use such info to insert a boot delay. _being a purist here_

Reply:
If it changes I'd expect it to change depending on the specific part number so I'd say DT is fine here (honestly I think hardcoding it could have been good enough). Don't mind renaming it and, let's go with yours.